src/WorkflowsRules/Contracts/Interfaces/ActionInterfaces.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Kanvas\Packages\WorkflowsRules\Contracts\Interfaces;
+
+interface ActionInterfaces
+{
+    public function handle(object $entity, array $params = []);
```

Comment:
@FredPeal instead of object it would be better to be a specify interface that we have to pass